### PR TITLE
[AL-4764] Rename Classification.instructions to Classification.name

### PR DESCRIPTION
**Pull request by whistler**

### CODE REVIEW

**Review by apollonin — APPROVED** [review]
look good to me!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Changelog
 
 # Version 3.37.0 (2023-xx-xx)
+## Changed
```
Comment (kkim-labelbox) [inline CHANGELOG.md:4]: @karenkyang @whistler are we okay with this being a breaking change?

File: labelbox/schema/ontology.py [hunk 2]
```diff
@@ -145,13 +147,26 @@ def __post_init__(self):
...
                 "no longer be able to be created in the Editor on 3/31/2022.")
 
-    @property
-    def name(self) -> str:
-        return self.instructions
+        if self.name is None:
+            msg = (
+                "When creating the Classification feature, please use “name” "
+                "for the classification schema name, which will be used when "
+                "creating annotation payload for MAL and Labelimport. "
```
Comment (kkim-labelbox) [inline labelbox/schema/ontology.py:154]: `creating annotation payload for Model-Assisted Labeling Import and Label Import`. Sorry for being a stickler. Just want to make sure we clean up any wording that goes out to customers

File: CHANGELOG.md (hunk 1 above)
Comment (kkim-labelbox) [reply CHANGELOG.md:4]: Ignore this. I saw that we're setting name = instruction in Classification class!

File: labelbox/schema/ontology.py (hunk 2 above)
Comment (whistler) [reply labelbox/schema/ontology.py]: I have updated the message.

**Review by kkim-labelbox — APPROVED** [review]

**Pull request merged**